src/app/api/sendgrid/route.js

```diff
@@ -0,0 +1,39 @@
+import { NextResponse } from "next/server";
```

Comment:
Why are the sendgrid code in mugglelink-form, and mlink-frontend?
They are duplicated. Can we make it once?
(Ideally in backend)

Reply:
since we have seperate email templates for both muggle form and mlink we should keep the email part seperated